frontend/components/commons/results/ResultsList.vue

```diff
@@ -16,32 +16,34 @@
   -->
 
 <template>
-  <div class="list">
+  <div class="content">  
     <slot name="header" />
```

Comment:
I just saw it. Could we use this header and remove the `results-header` slot?

Reply:
Not really, the header is placed at the top outside the scrolling part